docs/pages/router/reference/url-parameters.mdx

```diff
@@ -316,6 +316,26 @@ export default function User() {
 }
 ```
 
+{/* prettier-ignore */}
+{/*
+REMOVE COMMENT WHEN https://github.com/expo/expo/pull/30268 IS PUBLISHED
```

Comment:
```suggestion
TODO: REMOVE COMMENT WHEN https://github.com/expo/expo/pull/30268 IS PUBLISHED
```
Just a small suggestion. Let's add a Todo reference here if we're planning to keep this reference for some time.